### DISCUSSION TITLE
New Video Download Helper marked as Trojan by Acronis?

### DISCUSSION THREAD

**Discussion by Mike-vdh** [aclap-dev/video-downloadhelper#847] [Help and Q&A]
### Your question

Hi.
The VDP extension on Firefox wanted to be renewed, so I renewed it via Mozilla addons.
On trying to download a video now, the new extension tells me it is old, I have to renew (blue button "actualize"). 
So I click the renew button, I am led to "https://www.downloadhelper.net/install-coapp-v2"

If I now download the installer from there, and check this one with virustotal first, I receive:

1/69 security vendor and 1 sandbox flagged this file as malicious Reanalyze e1af498f95432a4f2f666a8cbec7bd6ab9deb4d1695c6e7cde7acb9bd408e600 vdhcoapp-windows-x86_64-installer.exe Security vendors' analysis VBA32 TrojanDownloader.Private Acronis (Static ML)

Any clue?
Thx

### Addon & CoApp version and details

Add on: Version
    9.0.2.3
Size
    1.33 MB
Last updated
    12 days ago (May 28, 2024)

Installer:
vdhcoapp-windows-x86_64-installer.exe

**Comment by GrampaWildWilly** [comment]
Click the link at the top of this page labelled Wiki.  Look for the word "antivirus" on that page.  Click that.  Read what you find.
